docs/install.md

```diff
@@ -3,35 +3,48 @@
 post installation to ensure the operator can function correctly.
 
-## STS Clusters
-
-### Post operator installation
+- [STS Clusters](#sts-clusters)
+    - [Post operator installation](#post-operator-installation)
+        - [Use CredentialsRequest created by the operator](#use-credentialsrequest-created-by-the-operator)
+        - [Use predefined secret](#use-predefined-secret)
+            - [Option 1. Using ccoctl](#option-1-using-ccoctl)
+            - [Option 2. Using AWS CLI](#option-2-using-aws-cli)
 
-In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
-the **cloud-credential-operator** and the manual intervention done by the
-cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
-`ccoctl` binary can be used to facilitate this task.
+## STS Clusters
+In an STS cluster, **cloud-credential-operator** do not automatically provision secrets for `CredentialsRequest`s
+and the manual intervention done by the cluster-admin is required.
+IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
 
 Normally, the **aws-load-balancer-operator** relies on the **cloud-credential-operator**
-to provision the secret for the operated controller using `CredentialsRequest`. And so in an STS cluster this
+to provision the secret for the operated controller using `CredentialsRequest`. However, in an STS cluster this
 secret needs to be provisioned manually. The **aws-load-balancer-operator** will wait until the required
 secret is created and available before spawning the **aws-load-balancer-controller** pod.
 
-#### Pre-Requisites
+### Post operator installation
+
+There are three ways to manually create the required IAM resources:
+- Using a `CredentialsRequest` generated by the operator for an existing `AWSLoadBalancerController`.
+- Using a pre-defined `CredentialsRequest` from this repository for an `AWSLoadBalancerController` that has not yet been created.
+- Employing AWS CLI commands for an `AWSLoadBalancerController` that has not yet been created.
+
+For handling `CredentialsRequests`, the cloud credential operator utility, [ccoctl](https://docs.openshift.com/container-platform/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts), can be utilized.
+If you prefer not to use `ccoctl` or are operating on a system not supported by it, the AWS CLI can be an alternative.
 
-#### [Extract and prepare the `ccoctl` binary](https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/cco-mode-sts.html#cco-ccoctl-configuring_cco-mode-sts)
+The generated credentials secret can either be awaited by the controller pod already or can be configured in the credentials field of the AWSLoadBalancerController prior to creating the controller.
```

Comment:
It's not clear what "awaited b the controller pod already" means here.

Reply:
Rephrased to

> The controller pod will wait for the generated credentials secret if `AWSLoadBalancerController` already exists or it can be configured in the `credentials` field of `AWSLoadBalancerController` before its creation.